resources/build/bootstrap-ubuntu-20.04.sh

```diff
@@ -39,7 +39,7 @@ conan profile update settings.compiler.libcxx=libstdc++ default
 # system packages are already available. In particular, this affects
 # recent versions of the xorg/system recipe (a dependency of cpython).
 # The problem is reported and fixed in https://github.com/conan-io/conan/pull/11712
-conan install --install-folder "$CONAN_USER_HOME" --build=missing \
+conan install --install-folder "$WORKSPACE/.conan" --build=missing \
```

Comment:
[question] What are peoples thoughts on this being hidden? Its a critical dir (eg. one you have to set with `--toolchain` etc), could it make more sense to suggest a workflow where it's visible? I don't know what the precedent is in general conan usage in the world at large.

Reply:
I guess I see it as a transient generated directory that's not part of the main source tree.  But happy to remove the `.` if that's the consensus.

Reply:
Same opinion as David, "feels" nicer as a hidden dir, but very un-strong opinions. I've never found a directory being hidden to be a point of confusion, personally.

Reply:
Cool beans.